server/client.go

```diff
@@ -195,6 +196,15 @@ func (client JiraClient) GetIssue(key string, options *jira.GetQueryOptions) (*j
 	return issue, nil
 }
 
+// GetWatchers returns an array of Jira watchers for a given issue.
```

Comment:
```suggestion
// GetWatchers returns an array of Jira users watching a given issue.
```

Reply:
The new text you've committed says "Jira users watching for a given issue.", but I think it should be "Jira users watching a given issue."